### ISSUE TITLE
window print opens after new Printd()

### ISSUE THREAD

**Issue by KedProgrammer** [joseluisq/printd#29]
## Expected Behavior
The print window should open only when  [variable].print()
...

## Actual Behavior
the window print opens after new Printd()
...

## Steps to Reproduce the Problem

  1. initialize a printD object
  1. ...
  1. ...

## Specifications

  - Version: `v1.0.1`
  - Platform: `v0.0.0`
  - Node version: `v10.12.0`
  - Browser version: `v70.0.3538.77`

**Comment by joseluisq** [comment]
[This code example](https://codepen.io/joseluisq/pen/VzRpGb) is working without problems using [v1.3.0](https://github.com/joseluisq/printd/releases/tag/v1.3.0).

Please, provide an example that reproduces your issue.

**Comment by joseluisq** [comment]
I will close this issue due to inactivity. Feel free to open it again if it's needed.

**Comment by saulshanabrook** [comment]
I have hit this bug, but only in Firefox. I just tried the example in firefox and it open a print dialogue on load.

**Comment by saulshanabrook** [comment]
I worked around this by only creating a printd instance when we need one to print, instead of globally:

```
  /**
   * We don't create the printd instance initially, because
   * doing so will trigger a false print on FireFox
   * 
   * See https://github.com/joseluisq/printd/issues/29
   */
  function getPrintd(): Printd {
    if (_PRINTD) {
      return _PRINTD;
    }
    _PRINTD = new Printd();
    return _PRINTD;
  }

  /**
   * Prints a widget by copying it's DOM node
   * to a hidden iframe and printing that iframe.
   */
  export function printWidget(widget: Widget) {
    getPrintd().print(widget.node);
  }
```

**Comment by joseluisq** [comment]
@saulshanabrook Thanks for sharing. I have confirmed it in Firefox too.
I think it's not a library's issue but rather something to take in mind before get an instance of Printd.

**Comment by joseluisq** [comment]
This issue was fixed in [v1.3.1](https://github.com/joseluisq/printd/releases/tag/v1.3.1) release.
